frame/elections-phragmen/src/lib.rs

```diff
@@ -116,6 +126,118 @@ type BalanceOf<T> =
 type NegativeImbalanceOf<T> =
 	<<T as Trait>::Currency as Currency<<T as frame_system::Trait>::AccountId>>::NegativeImbalance;
 
+/// Migrations to version [`3.0.0`], as denoted by the changelog.
+pub mod migrations_3_0_0 {
+	use super::*;
+	use frame_support::{
+		migration::StorageKeyIterator,
+		traits::{GetPalletVersion, PalletVersion},
+		Twox64Concat,
+	};
+
+	pub fn apply<T: Trait>(
+		old_voter_bond: BalanceOf<T>,
+		old_candidacy_bond: BalanceOf<T>,
+	) -> Weight {
+		let current_version = <Module<T> as GetPalletVersion>::current_version();
+		let maybe_storage_version = <Module<T> as GetPalletVersion>::storage_version();
+
+		if current_version == PalletVersion::new(3, 0, 0) {
+			match maybe_storage_version {
+				Some(storage_version) if storage_version == PalletVersion::new(2, 0, 0) => {
```

Comment:
chain on substrate 2.0 don't have version 2.0.0 in storage (storage version wasn't introduced yet), maybe we should do `<=` instead then. Or document it.